Exercise 1/Start/Clock/ViewControllers/AboutViewController.cs

```diff
@@ -17,9 +18,24 @@ public override void ViewDidLoad ()
 
 			EdgesForExtendedLayout = UIRectEdge.None;
 
-			string aboutUrl = NSBundle.MainBundle.BundlePath + "/About.html";
+			var aboutUrl = NSUrl.FromFilename(NSBundle.MainBundle.BundlePath + "/About.html");
+			AboutWebView.NavigationDelegate = this;
+			AboutWebView.LoadFileUrl(aboutUrl, aboutUrl.RemoveLastPathComponent());
+		}
 
-			AboutWebView.LoadRequest (new NSUrlRequest (new NSUrl (aboutUrl, false)));
+		[Export("webView:decidePolicyForNavigationAction:decisionHandler:")]
+		public void DecidePolicy(WKWebView webView, WKNavigationAction navigationAction, Action<WKNavigationActionPolicy> decisionHandler)
```

Comment:
@Krumelur Do you mind walking me through this a little? Where does the `Action<WKNavigationActionPolicy>` parameter come from here?